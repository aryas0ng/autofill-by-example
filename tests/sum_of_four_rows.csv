11,33
15,25
2,12
5,14
3,52
2,
4,
43,
5,
6,
9,
3,
4,
2,
5,
2,
6,
1,
8,
6,
1,
21,
3,