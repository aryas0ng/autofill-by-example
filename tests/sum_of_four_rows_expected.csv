11,33
15,25
2,12
5,14
3,52
2,54
4,58
43,63
5,23
6,22
9,18
3,14
4,13
2,15
5,14
2,17
6,21
1,16
8,36
6,31
1,25
21,24
3,3